### PR TITLE
Contribution: Some German Translation

### PR DESCRIPTION
**Pull request by justuser668**

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Books/first-steps-tutorials.json [hunk 1]
```diff
@@ -208,11 +248,6 @@
             "text": "Foundations->Concept->List of Plugin Workspaces and Tutorials->Content",
             "updated": 1610955413534
         },
-        {
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Books/first-steps-tutorials.json:211]: Was it intended byyou to remove that part?
@justuser668 ustuser668

Comment (justuser668) [reply Projects/Foundations/Schemas/Docs-Books/first-steps-tutorials.json:211]: was a fail, is changed in newer translations

**Comment by gitforkgit** [comment]
Alright, i will go trough it :+1:

File: Projects/Foundations/Schemas/Docs-Books/first-steps-tutorials.json (hunk 1 above)
Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Books/first-steps-tutorials.json:211]: For me it doesnt seem like this part changed,did you app.update after fixing the problem?

Comment (justuser668) [reply Projects/Foundations/Schemas/Docs-Books/first-steps-tutorials.json:211]: jetzt aber, entschuldige war gestern ein unbedachter Mausklick...

**Comment by Luis-Fernando-Molina** [comment]
This Pull Request could not be automatically merged and was closed by the Superalgos Governance System because it was detected that a User Profile file... 

https://github.com/Superalgos/Superalgos/raw/8aee90a6c5a73991e0c32ac3b2d421641f3f9a94/Projects/Governance/Plugins/User-Profiles/justuser668.json

...was submitted together with  44 other file/s. User Profiles files as per the Governance System rules, must be the only file present at a Pull Request in order to pass all the validations and be automatically merged. 

 If you intentionally submitted the extra files, the way to get your work merged is by first removing your User Profile file from the Plugins folder at your local installation and executing an app.contribute again. Then wait until the PR is merged (manually reviewed) and after that you put your User Profile file again and create a new PR only with that file with an app.contribute. That last PR should be merged automatically.

**Pull request closed without merging**